docs/generated/stablehlo_passes.md

```diff
@@ -338,6 +338,59 @@ Modules valid for shape refinement must have the following properties:
   * All calls to a single function resolve to the same argument shapes, and no
     recursive / co-recursive function calls are made.
 
+### `-stablehlo-wrap-in-composite`
+
+_Wraps a non-composite  StableHLO op in a composite op._
+
+Wraps StableHLO ops, as specified by the pass option flag, in a
```

Comment:
Let's talk about the design more. It's missing a few things to be generally useful:

- What to do with ops that have attributes?
- What should the composites be named, or should the name be user specified as a part of the callback fn, etc

Need to think more, maybe we have a little brainstorm and run through some examples

Reply:
All good points! SGTM.

>> What to do with ops that have attributes?

All the op attributes are preserved as composite attributes.  To demonstrate that I have used `convolution  op` as an example in `stablehlo/tests/transforms/stablehlo_wrap_in_composite.mlir`.